docs/reference/bpmn-processes/manual-tasks/manual-tasks.md

```diff
@@ -0,0 +1,23 @@
+---
+id: manual-tasks
+title: "Manual tasks"
+---
+
+A manual task defines a task that is external to the BPM engine. It is used to model work that is done
+by somebody who the engine does not need to know of and there is no known system or UI interface. 
+For the engine, a manual task is handled as a pass-through activity, automatically continuing the 
+process at the moment the process execution arrives at it.
+
+![task](assets/manual-task.png)
```

Comment:
:wrench: The image of the BPMN element is too big :sweat_smile: 

![Screenshot from 2021-09-16 06-08-15](https://user-images.githubusercontent.com/4305769/133547902-d32d4179-c227-4bdb-a7c5-509ce9a0d620.png)

The image has a significantly bigger size compared to other pages. We don't have a fixed rule set for images but we could try to align it.

![Screenshot from 2021-09-16 06-08-27](https://user-images.githubusercontent.com/4305769/133547905-afcfbe95-c6be-4728-9358-ada4f84775fe.png)